src/pybind/mgr/dashboard/controllers/nfsganesha.py

```diff
@@ -279,22 +281,22 @@ def fsals(self):
     @Endpoint('GET', '/lsdir')
     def lsdir(self, root_dir=None, depth=1):
         if root_dir is None:
-            root_dir = "/"
+            root_dir = os.sep
```

Comment:
Since the target is CephFS, the separator will always be "/". While I don't ever see the ceph-mgr running on Windows, if it did, this code would not work.

Reply:
What about https://docs.python.org/3/library/os.html#os.sep? ``os.sep`` takes care of the platform it is running on.

Reply:
I'd keep it simple and just use `/`

Reply:
Done